build.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
-javac -cp lib/junit-4.12.jar:lib/hamcrest-core-1.3.jar *.java
-java -cp .:lib/junit-4.12.jar:lib/hamcrest-core-1.3.jar org.junit.runner.JUnitCore EdgeConnectorTest
-java RunEdgeConvert
+javac -cp lib/junit-4.12.jar:lib/hamcrest-core-1.3.jar -d build src/java/*.java test/java/EdgeConnectorTest.java 
```

Comment:
FYI it would be better to use `test/java/*.java` instead of hardcoding the class name, because what will happen when we add another test class